docker/Makefile

```diff
@@ -1,14 +1,29 @@
+DB_IP=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' db`
+DB_PASS="NotSoSecretPassword"
+
 build:
 	docker build -t faf-image -f Dockerfile ../
 
 build_local:
 	docker build -t faf-image-local -f Dockerfile_local ../
 
+build_db:
+	docker build -t postgres_w_semver -f Dockerfile_db ../
+
 run:
-	docker run --name faf -dit faf-image
+	docker run --name faf -dit -e PGHOST=$(DB_IP) -e PGUSER=faf -e PGPASSWORD=scrt -e PGPORT=5432 -e PGDATABASE=faf -p 8080:8080 faf-image
 
 run_local:
-	docker run --name faf -dit faf-image-local
+	docker run --name faf -dit -e PGHOST=$(DB_IP) -e PGUSER=faf -e PGPASSWORD=scrt -e PGPORT=5432 -e PGDATABASE=faf -p 8080:8080 faf-image-local
+
+# mkdir /var/tmp/data
+# chown 26:26 /var/tmp/data
+# chcon -t svirt_sandbox_file_t /var/tmp/data
```

Comment:
Either it is noop and remove it. Or if this is meant as "sometimes this can be handy" then please put comment above this which state when it is handy.